README.md

```diff
@@ -22,8 +22,7 @@ Create a new datasource with a name and select `type` as `MetaQueries`
 ## Examples
 #### Arithmetic
 Lets you perform arithmetic operations on one or more existing queries.
-![Screenshot](https://raw.githubusercontent.com/GoshPosh/grafana-meta-queries/master/img/arithmetic-ex1.png?raw=true "Arithmetic Example 1 - Metric * 2")
-![Screenshot](https://raw.githubusercontent.com/GoshPosh/grafana-meta-queries/master/img/arithmetic-ex2.png?raw=true "Arithmetic Example 2 - Metric A + Metric B")
+![Screenshot](https://github.com/sunnut/grafana-meta-queries/blob/master/img/arithmetic-ex.jpg?raw=true "Arithmetic Example - Metric A + Metric B")
```

Comment:
not sure why would we remove existing examples ?

Reply:
I didn't even see this was changed from the previous merge request. I reverted these changes (I may add the last screenshot later, but with the light theme for consistency)